justfile

```diff
@@ -40,3 +44,8 @@ format-terraform:
 validate-terraform:
   if [ -z "${terraform}" ]; then echo "ERROR: please install terraform or opentofu"; exit 1; fi
   set -e; for repo in */; do (cd "$repo" && echo "Processing ${repo%/}..." && $terraform init -upgrade -reconfigure && $terraform validate) || exit 1; done
+
+[group("Command")]
```

Comment:
I would leave this ungrouped rather than having a "Command" group which is too generic